README.md

```diff
@@ -90,7 +92,16 @@ Listing of useful (mostly) public learning resources for machine learning applic
 - [Deep Learning and Reinforcement Learning Summer School, Toronto, Canada](https://dlrlsummerschool.ca/) (July 25 - August 3, 2018)
 - [PAISS: Artificial Intelligence Summer School, Grenoble, France](https://project.inria.fr/paiss/) (July 2-6, 2018)
 
+##### Upcoming:
+
+- [Machine Learning Summer School 2018, Madrid, Spain](http://mlss.ii.uam.es/mlss2018/index.html) (August 27 - September 7, 2018)
```

Comment:
It seems there's a duplication with the section above. Do you have time to clean that up? If not I can get it.

Reply:
Thanks for pointing it out. Fixed in latest commit!